Mage.Sets/src/mage/cards/a/AshesOfTheFallen.java

```diff
@@ -62,7 +64,12 @@ public boolean apply(Game game, Ability source) {
                 for (UUID cardId : controller.getGraveyard()) {
                     Card card = game.getCard(cardId);
                     if (card != null && card.isCreature(game) && !card.hasSubtype(subType, game)) {
```

Comment:
this one also needs update to check creature on part, not main card (and pretty sure no reason to check if already has the subtype)